src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input } from './style';
+
+function TextField(props) {
+  console.log('--------Inside TextField Constructor---------', props);
+  const { value, disabled, error } = props;
+  return (
+    <div>
+      <Input type="text" value={value} disabled={disabled} />
+      <p style={{ color: 'red' }}>{error}</p>
```

Comment:
use color value from style.js and only shows error tag if error is occured

Reply:
Done